modules/psap-special-resource-operator-using.adoc

```diff
@@ -0,0 +1,380 @@
+  name: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+  annotations:
+    specialresource.openshift.io/wait: "true"
+    specialresource.openshift.io/state: "driver-container"
+    specialresource.openshift.io/driver-container-vendor: simple-kmod
+    specialresource.openshift.io/kernel-affine: "true"
+spec:
+  updateStrategy:
+    type: OnDelete
+  selector:
+    matchLabels:
+      app: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+  template:
+    metadata:
+      # Mark this pod as a critical add-on; when enabled, the critical add-on scheduler
+      # reserves resources for critical add-on pods so that they can be rescheduled after
+      # a failure.  This annotation works in tandem with the toleration below.
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+      labels:
+        app: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+    spec:
+      serviceAccount: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+      serviceAccountName: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+      containers:
+      - image: image-registry.openshift-image-registry.svc:5000/{{.Values.specialresource.spec.namespace}}/{{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}:v{{.Values.kernelFullVersion}}
+        name: {{.Values.specialresource.metadata.name}}-{{.Values.groupName.driverContainer}}
+        imagePullPolicy: Always
+        command: ["/sbin/init"]
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@{{.Values.specialresource.metadata.name}}"]
+        securityContext:
+          privileged: true
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+        feature.node.kubernetes.io/kernel-version.full: "{{.Values.KernelFullVersion}}"
+----
+
+.. Change into the `chart/simple-kmod-0.0.1` directory
++
+[source, terminal]
+----
+cd ..
+----
+
+.. Save the following YAML for the Chart as `Chart.yaml` in the `chart/simple-kmod-0.0.1` directory.
++
+[source, yaml]
+----
+apiVersion: v2
+name: simple-kmod
+description: Simple kmod will deploy a simple kmod driver-container
+icon: https://avatars.githubusercontent.com/u/55542927
+type: application
+version: 0.0.1
+appVersion: 1.0.0
+----
+
+. Now that the structure for the Helm chart is prepared, create the chart with the helm package command, from the `chart` directory:
++
+[source,terminal]
+----
+cd ..
+helm package simple-kmod-0.0.1/
+----
++
+.Example output
+[source,terminal]
+----
+Successfully packaged chart and saved it to: /data/dagray/git/dagrayvid/special-resource-operator/yaml-for-docs/chart/simple-kmod-0.0.1/simple-kmod-0.0.1.tgz
+----
+
+. Create a ConfigMap to store the chart files
++
+[source,terminal]
+----
+mkdir cm
```

Comment:
Separate multiple commands into separate blocks. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output